tests/unit/functions/server_functions_spec.js

```diff
@@ -0,0 +1,162 @@
+const { cors } = require("../../../js/server_functions");
+const nodeVersion = process.version.match(/^v(\d+)\.*/)[1];
+
+describe("server_functions tests", () => {
+	describe("The cors method", () => {
+		let fetchResponse;
+		let fetchResponseHeadersGet;
+		let fetchResponseHeadersText;
+		let corsResponse;
+		let request;
+
+		let nodefetch;
+		if (nodeVersion < 18) {
```

Comment:
Now that we reverted the native fetch implementation (https://github.com/MichMich/MagicMirror/pull/2961) do we still need this?

Reply:
No, implemented everything for `node-fetch` now. Almost sad after finally getting the tests to pass for all node versions 😅